diff --git a/dist/index.mjs b/dist/index.mjs
index 10df438c1edbe5f8952990a3af14739b8c01ee5f..63656aeeba32b3f927f2dcfcf192d75527ef8005 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -233,24 +233,24 @@ function presetTypography(options) {
             return;
           const colorObject = typeof baseColor === "object" ? baseColor : {};
           return {
-            "--un-prose-body": colorObject[700] ?? baseColor,
-            "--un-prose-headings": colorObject[900] ?? baseColor,
-            "--un-prose-links": colorObject[900] ?? baseColor,
-            "--un-prose-lists": colorObject[400] ?? baseColor,
-            "--un-prose-hr": colorObject[200] ?? baseColor,
-            "--un-prose-captions": colorObject[500] ?? baseColor,
-            "--un-prose-code": colorObject[900] ?? baseColor,
-            "--un-prose-borders": colorObject[200] ?? baseColor,
-            "--un-prose-bg-soft": colorObject[100] ?? baseColor,
-            "--un-prose-invert-body": colorObject[200] ?? baseColor,
-            "--un-prose-invert-headings": colorObject[100] ?? baseColor,
-            "--un-prose-invert-links": colorObject[100] ?? baseColor,
-            "--un-prose-invert-lists": colorObject[500] ?? baseColor,
-            "--un-prose-invert-hr": colorObject[700] ?? baseColor,
-            "--un-prose-invert-captions": colorObject[400] ?? baseColor,
-            "--un-prose-invert-code": colorObject[100] ?? baseColor,
-            "--un-prose-invert-borders": colorObject[700] ?? baseColor,
-            "--un-prose-invert-bg-soft": colorObject[800] ?? baseColor,
+            "--un-prose-body": colorObject[11] ?? baseColor,
+            "--un-prose-headings": colorObject[12] ?? baseColor,
+            "--un-prose-links": colorObject[12] ?? baseColor,
+            "--un-prose-lists": colorObject[10] ?? baseColor,
+            "--un-prose-hr": colorObject[8] ?? baseColor,
+            "--un-prose-captions": colorObject[10] ?? baseColor,
+            "--un-prose-code": colorObject[12] ?? baseColor,
+            "--un-prose-borders": colorObject[6] ?? baseColor,
+            "--un-prose-bg-soft": colorObject[2] ?? baseColor,
+            "--un-prose-invert-body": colorObject[11] ?? baseColor,
+            "--un-prose-invert-headings": colorObject[12] ?? baseColor,
+            "--un-prose-invert-links": colorObject[12] ?? baseColor,
+            "--un-prose-invert-lists": colorObject[10] ?? baseColor,
+            "--un-prose-invert-hr": colorObject[8] ?? baseColor,
+            "--un-prose-invert-captions": colorObject[10] ?? baseColor,
+            "--un-prose-invert-code": colorObject[12] ?? baseColor,
+            "--un-prose-invert-borders": colorObject[6] ?? baseColor,
+            "--un-prose-invert-bg-soft": colorObject[2] ?? baseColor,
             "--un-prose-font-mono": theme.fontFamily?.mono
           };
         },
